elektronLogging/clsfnTaskDistOverheadLogger.go

```diff
@@ -0,0 +1,50 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+	"strings"
+)
+
+type ClsfnTaskDistOverheadLogger struct {
```

Comment:
Maybe refactor to `ClsfnTaskDistrOverheadLogger` so that it is clear that we are intending "Distribution" and now "Distance"?

Reply:
Done